dynawo/CMakeLists.txt

```diff
@@ -225,27 +225,19 @@ endif()
 # Tell BOOST to disable auto linking
```

Comment:
The set(LIBXML_HOME  ...) is missing in the list at line 45

Reply:
1 - I don't understand how LIBXML2_HOME may be used if I set it, and who will use it.
2 - What will be the content of the variable ? library is in /usr/lib by example and include file /usr/include/libxml2. the finder gives us 2 variables LIBXML2_LIBRARY and LIBXML2_INCLUDE_DIR. I did not find out yet the the purpose of a third one.
3 - Why should I call it LIBXML2_HOME since it is some times LIBXML2_ROOT that may be used? That is not clear yet for me.